app/code/Transoft/Blog/view/frontend/templates/singleBlogTmpl.phtml

```diff
@@ -0,0 +1,25 @@
+<?php
```

Comment:
Please rename template file to **single_blog_post.phtml**

Reply:
fixed